docs/reference/compatibility.txt

```diff
@@ -531,18 +540,26 @@ provided compatibility with wherever possible.
    * - ``ActiveModel``
      - |checkmark|
 
-   * - ``ActiveRecord``
-     - |x|
-
    * - ``ActiveStorage``
      - |x| [#rails-activerecord-dependency]_
 
-.. [#rails-activerecord-dependency] Depends on ``ActiveRecord``
+.. [#rails-actioncable-dependency] There is currently no MongoDB adapter for
+   ``ActionCable``, however any existing adapter can be used successfully in conjunction
+   with Mongoid models
+
+.. [#rails-activerecord-dependency] Depends directly on ``ActiveRecord``
+
+.. [#rails-activejob-dependency] Serialization of BSON & Mongoid objects  works best
+   if you explicitly send ``ObjectId``'s as strings, and reconstitute them in the job:
+
+   .. code-block:: ruby
 
-.. [#rails-activejob-dependency] Serialization of BSON & Mongoid objects is
-   fraught, works best if you explicitly send ``ObjectId``'s as strings, and
-   reconstitute them in the job
+    record = Model.find(...)
+    MyJob.perform_later(record._id.to_s)
 
-.. [#rails-actioncable-dependency] not all subscription adapters are compatible
-   with Mongoid (e.g. the PostgreSQL adapter) but most if not all other adapters
-   are compatible.
+    class MyJob < ApplicationJob
+      def perform(id_as_string)
+        id = BSON::ObjectId.from_string(id_as_string)
+        record = Model.find(id) # ...
```

Comment:
You can pass `id` a string directly into `Model.find`, so the `id = BSON...` line is not needed.